staging/src/k8s.io/kube-scheduler/config/v1beta1/types_pluginargs.go

```diff
@@ -172,6 +172,19 @@ type NodeResourcesMostAllocatedArgs struct {
 	Resources []ResourceSpec `json:"resources,omitempty"`
 }
 
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+
+// NodeResourcesBalancedAllocationArgs holds arguments used to configure NodeResourcesBalancedAllocation plugin.
+type NodeResourcesBalancedAllocationArgs struct {
+	metav1.TypeMeta `json:",inline"`
+
+	// Resources to be managed, if no resource is provided, default resource is
+	// "cpu" and "memory".
+	// Weight should be always set to 1 per the nature of the scroing plugin.
+	// +listType=atomic
```

Comment:
It should be map https://github.com/kubernetes/kube-openapi/blob/master/pkg/idl/listtype_test.go#L28-L29

Reply:
Done